mapengine/worldwind/aar/build.gradle

```diff
@@ -3,6 +3,7 @@ apply from: "$rootDir/build-support/emp3-gradle-plugin/src/main/resources/androi
 dependencies {
     compile          ("com.android.support:support-annotations")
     provided project (":emp3-android-sdk")
+    provided files ("$rootDir/3rdpartylibrary/cmapi/cmapi.jar")
```

Comment:
Could be worth putting a version number on the cmapi jar? Other than that everything looks good to me, approving.